crates/iota-keys/src/keystore.rs

```diff
@@ -249,6 +251,13 @@ impl AccountKeystore for FileBasedKeystore {
         Ok(())
     }
 
+    fn remove_key(&mut self, address: &IotaAddress) -> Result<(), anyhow::Error> {
+        self.aliases.remove(address);
+        self.keys.remove(address);
+        self.save()?;
```

Comment:
Do we also need to unset the active address here?

Reply:
Good point. Just unset, or reset it to something else?

Reply:
Not sure how the logic is built around it, if it would automatically set a new one the next time we try to get it, then just unsetting it should be fine

Reply:
I did it somewhere else because keystore has no concept of active address. Also confirmed that it already sets it to something else automatically.
https://github.com/iotaledger/iota/pull/7151/commits/73dbeda5780290a6d969c798ff710f893de07969